app/src/main/java/com/gamr/gamr/ConversationActivity.java

```diff
@@ -0,0 +1,125 @@
+package com.gamr.gamr;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.v7.app.ActionBarActivity;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import com.gamr.gamr.ServerRepresentations.Message;
+
+import java.util.List;
+
+/**
+ * An activity that represents a conversation between two people that allows for the sending and
+ * viewing of messages.
+ */
+public class ConversationActivity extends ActionBarActivity {
+    public static final String SENDER_KEY = "MESSAGE_SENDER";
+    private List<Message> mMessageList;
+    private ConversationAdapter mAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_conversation);
+
+
+        // First we find the list view
+        final ListView messagesListView = (ListView) findViewById(R.id.conversationListView);
+
+        // From there we will populate our list with our current messages
+        if (mMessageList == null) {
+            // TODO We should obviously change how the two activities communicate with each other
+            // I am thinking we could have some global object that will track the messaging
```

Comment:
On the detailed design, I was thinking originally of a global current user object? That way, we can just do something like: 

CurrentUser.getConversation(getIntent().getExtras().getString(SENDER_KEY));

where SENDER_KEY is the other user.